### PR TITLE
add stars-task solution

### PR DESCRIPTION
**Pull request by BohdanDymydiuk**
- [DEMO LINK](https://BohdanDymydiuk.github.io/layout_stars/)
- [TEST REPORT LINK](https://BohdanDymydiuk.github.io/layout_stars/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  background-image: url(images/star-active.svg);
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+.stars--5 :nth-child(n) {
+  background-image: url(images/star-active.svg);
+}
```
Comment (anastasiiavorobiova) [inline src/style.css:40]:
```suggestion

```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 :nth-child(-n+1) {
```
Comment (anastasiiavorobiova) [inline src/style.css:22]:
```suggestion
.stars--1 :nth-child(-n+1),
.stars--2 :nth-child(-n+2),
.stars--3 :nth-child(-n+3),
.stars--4 :nth-child(-n+4),
.stars--5 :nth-child(n) {
```

Comment (anastasiiavorobiova) [reply src/style.css:22]: You can use commas to avoid repeating code

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done

File: src/style.css (hunk 2 above)
Comment (BohdanDymydiuk) [reply src/style.css]: Yeah, thanks. I forgot that I can't combine selectors, but I can use commas

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome work!

**Pull request closed without merging**